client_handler.cpp

```diff
@@ -353,6 +353,8 @@ void ClientHandler::OnBeforeContextMenu(CefRefPtr<CefBrowser> browser,
 		model->Remove(commandId);
 	}
 	removeCommandIds.clear();
+	bool needReload = true;
+	bool needPrint = true;

```

Comment:
It might give a strange impression because needXXX  requires operations as if the existence of additional processing implementation.

I mean:

```
needReload = true
if (needReload) {
   // do Reload operation here
}
```

Actually it just control show/hidden context menu, so there is more meaningful/appropriate naming.

```
showReloadContextMenu = true;
showPrintContextMenu  = true;
```

or AddReloadContextMenu or something else.

Reply:
I see.

I felt containing `ContextMenu` is somehow redundant because this method name is `OnBeforeContextMenu` and it is clear that we are handling context menu.
As a result, I have renamed `needReload` to `addReloadItem` and `needPrint` to `addPrintItem`.